### PR TITLE
Add test and comments for utils.py methods

### PR DESCRIPTION
**Pull request by sophietheis**
Add aspect ratio calculation function. 
Modify _to_2d an _to_3d function -> it calls to_nd function. 
Add comments for all function in utils.py except for combine_specs and set_data_columns.

### CODE REVIEW

**Comment by sophietheis** [comment]
Well, I have done something that make no error, but I'm not really sure about the result of AR. 
I tell myself that maybe I have to use np.svd method and not use face_projeted_pos.

File: tyssue/utils/utils.py [hunk 1]
```diff
@@ -237,27 +245,17 @@ def ar_calculation(sheet):
...
-        if minor == 0:
-            ar.append(0)
-        else:
-            ar.append(major / minor)
-    return pd.Series(ar)
+    from ..geometry.sheet_geometry import SheetGeometry
+
+    projected_srce_pos = pd.DataFrame()
+    for face in np.unique(sheet.edge_df['face']):
+        projected_srce_pos = SheetGeometry.face_projected_pos(sheet, face)
```
Comment (glyg) [inline tyssue/utils/utils.py]: I don't think this is what you intended. The face_projected_pos should go in the _compute_ar function to be called for each face, you can get the face index with df['face'].iloc[0].

File: tests/utils/test_utils.py [hunk 2]
```diff
@@ -33,18 +75,29 @@ def test_modify():
...
+
+def test_ar_calculation():
+    sheet = Sheet('test', *three_faces_sheet())
+    e0 = sheet.edge_df.index[0]
+    face = sheet.edge_df.loc[e0, 'face']
+    sheet.edge_df = sheet.edge_df.loc[sheet.edge_df.index[1:]].copy()
+    close_face(sheet, face)
+
+    sheet.face_df['AR'] = utils.ar_calculation(sheet)
+    assert 'AR' in sheet.face_df
```
Comment (glyg) [inline tests/utils/test_utils.py]: Maybe a better test here would be to deform the input sheet and asserting that the aspect ratio is coherent

Initialy, the three hexagonal sheets have an aspect ratio of (I think) $\sqrt{3}$. If you do something like `sheet.vert_df['x'] *= 2`, the ar will change, you can test that both before and after the change, the ar is consistent. In practice, you can compute it in a notebook to get the values before writing the test..

File: tyssue/behaviors/sheet_events.py [hunk 3]
```diff
@@ -19,6 +22,8 @@ def __init__(self, sheet, model, geom):
         self.sheet = sheet
         self.model = model
         self.geom = geom
+        self.current_deque = deque()
```
Comment (glyg) [inline tyssue/behaviors/sheet_events.py]: I'm going to rename these from `current_deque`, `next_deque` to just`current`, `next`

Comment (glyg) [inline tyssue/behaviors/sheet_events.py]: I'm going to rename these from `current_deque`, `next_deque` to just`current`, `next`

**Pull request merged**